app-doc/kicad-doc/kicad-doc-4.0.2.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Any reason why you don't use EAPI=6? Not using the latest EAPI creates more work on the long run.